core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -150,6 +150,37 @@ boolean testClassLoad(final String className, final String asTypeName)
    */
   void waitForBalance() throws AccumuloException;
 
+  /**
+   * Fails a fate transaction based on the given txID. At least one txID must be provided.
+   *
+   * @param txids
+   *          Transaction IDs to fail.
+   * @since 2.1.0
+   */
+  void fateFail(List<String> txids) throws AccumuloException;
+
+  /**
+   * Deletes a fate transaction based on the given txID. At least one txID must be provided.
+   *
+   * @param txids
+   *          Transaction IDs to delete.
+   * @since 2.1.0
+   */
+  void fateDelete(List<String> txids) throws AccumuloException;
+
+  /**
+   * Gathers Transaction status information for either all fate transactions or requested txIDs.
+   *
+   * @param txids
+   *          Transaction IDs to use as a filter. Optional.
+   * @param tStatus
+   *          Parsed TStatus for print filter. Optional.
+   * @return A list of TransactionStatues for corresponding txids
+   * @since 2.1.0
+   */
+  List<TransactionStatus> fateStatus(List<String> txids, List<String> tStatus)
+      throws AccumuloException;
+
```

Comment:
I think these are good so far but we need a way to get the `TxIds`. We need a method like this on InstanceOperations:
`List<TransactionStatus> fatePrint() throws AccumuloException;`

Here is an example of how it could be used:
<pre>
var list = client.instanceOperations().fatePrint();
for (var ts : list) {
  client.instanceOperations().fateFail(List.of(ts.getTxid()));
}
</pre>

I also think this would be a good opportunity to use a stronger type for Txid. This was discussed recently here: https://github.com/apache/accumulo/issues/2473#issuecomment-1039473784

Reply:
If this would benefit from some of the more aggressive FaTE refactoring you're working on that might go in 3.0, I'd be okay with pushing this PR off until then. But, otherwise, I'd generally agree with your comment above that this would be good to get into 2.1 (based on the intended outcome, not the code itself, since I haven't yet had a chance to review the code).